Gemfile

```diff
@@ -53,9 +53,7 @@ group :development, :test do
   gem 'toxiproxy', '~> 0.1.3'
 end
 
-group :development do
-  gem 'rails-erd'
-end
+gem 'rails-erd', group: :development
```

Comment:
It is used by https://github.com/rubygems/rubygems.org/blob/071df7d625ee18be64d225e78abb445fdbf17462/lib/tasks/gen_erd.rake